test/integration/proxyless/proxyless-demandA.test.ts

```diff
@@ -0,0 +1,68 @@
+import { describe, beforeEach, afterEach, it } from 'mocha'
+import { Express } from 'express'
+import { expect } from 'chai'
+
+import Indexer from '../../../src/lib/indexer/index.js'
+import { post } from '../../helper/routeHelper.js'
+import { seed, cleanup, parametersAttachmentId } from '../../seeds/onchainSeeds/demandA.seed.js'
+
+import { selfAddress, withIdentitySelfMock } from '../../helper/mock.js'
+import Database from '../../../src/lib/db/index.js'
+import ChainNode from '../../../src/lib/chainNode.js'
+import { pollTransactionState, pollDemandState } from '../../helper/poll.js'
+import { withAppAndIndexer } from '../../helper/chainTest.js'
+import { container } from 'tsyringe'
+
+describe('on-chain', function () {
+  this.timeout(80000)
+  const db = new Database()
+  const node = container.resolve(ChainNode)
```

Comment:
can I recommend we have an `it` that resolves the env and asserts we're not using a proxy.